habitat_sim/agent/agent.py

```diff
@@ -191,14 +191,30 @@ def get_state(self) -> AgentState:
         return state
 
     def set_state(
-        self, state: AgentState, reset_sensors: bool = True, is_initial: bool = False
+        self,
+        state: AgentState,
+        reset_sensors: bool = True,
+        infer_sensor_states: bool = True,
+        is_initial: bool = False,
     ):
         r"""Sets the agents state
 
         :param state: The state to set the agent to
         :param reset_sensors: Whether or not to reset the sensors to their
-            default intrinsic/extrinsic parameters before setting their
-            extrinsic state
+            default intrinsic/extrinsic parameters before setting their extrinsic state.
+
+            Setting this to `False` allows the agent base state to be moved and the new
+            sensor locations inferred without changing the configuration of the sensors
+            with respect to the base state of the agent.
+
+            Default: `True`
```

Comment:
The default is [shown in the docs already](https://aihabitat.org/docs/habitat-sim/habitat_sim.agent.Agent.html#set_state) so I don't think it's needed to be repeated here. Same below.